tsconfig-typecheck-jsdoc.json

```diff
@@ -16,6 +16,7 @@
     "noImplicitThis": true,
     "noPropertyAccessFromIndexSignature": false,
     "target": "ESNext",
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "skipLibCheck": true,
```

Comment:
:sweat_smile: look at the commit desc for more info.
Anyway, we need to merge #948 before we merge this one.

TODO @florian-sanders-cc remove this commit once #948 is merged.